deb/common/rules

```diff
@@ -21,6 +21,8 @@ override_dh_auto_install:
 	install -D -m 0755 /go/src/github.com/docker/cli/build/docker debian/docker-ce-cli/usr/bin/docker
 	# docker-ce install
 	install -D -m 0644 /sources/docker.service debian/docker-ce/lib/systemd/system/docker.service
+	install -D -m 0644 /sources/docker.socket debian/docker-ce/lib/systemd/system/docker.socket
+	install -D -m 0644 /sources/00-socket-activation.conf debian/docker-ce/etc/systemd/system/docker.service.d/00-socket-activation.conf
```

Comment:
i think this can be in a drop-in directory `/lib/systemd/system/docker.service.d/` instead of the `/etc/systemd/system/docker.service.d/` dir because the `/etc` dir is for admin override of package's systemd files that are put in `/lib`